apps/fluent-tester/src/FluentTester/FluentTester.tsx

```diff
@@ -204,16 +204,37 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
     );
   };
 
-  return (
-    <RootView style={themedStyles.root}>
-      {enableSinglePaneView ? <MobileHeader /> : <Header />}
+  const TesterContent: React.FunctionComponent<{}> = () => {
+    return (
+      <RootView style={themedStyles.root}>
+        {enableSinglePaneView ? <MobileHeader /> : <Header />}
 
-      <HeaderSeparator />
+        <HeaderSeparator />
 
-      <View style={fluentTesterStyles.testRoot}>
-        {enableSinglePaneView ? <MobileTestList /> : <TestList />}
-        {enableSinglePaneView ? <MobileTestComponentView /> : <TestComponentView />}
-      </View>
-    </RootView>
-  );
+        <View style={fluentTesterStyles.testRoot}>
+          {enableSinglePaneView ? <MobileTestList /> : <TestList />}
+          {enableSinglePaneView ? <MobileTestComponentView /> : <TestComponentView />}
+        </View>
+      </RootView>
+    );
+  };
+
+  const Win32TesterContent: React.FunctionComponent<{}> = () => {
+    return (
+      <RootView style={themedStyles.root}>
+        <FocusTrapZone>
```

Comment:
Could RootView be a FocusTrapZone? 
Related: Was the commits in which I refactored FluentTester for mobile (introducing RootView and the like) be when the regression happened?

Reply:
Dumb question: Why not just a FocusZone?

Reply:
I just created a branch off of 9a6fb7a97c6aa0f8cea946f1d9963bab027eebc6 which is the last commit before your change and I still get the same issue. I skipped running `yarn` and just did `yarn build` under apps, then `yarn bundle` under apps\win32, and `yarn run-win32` to pull the tester up. That should be the right steps to take to test a previous state of the app, right? Just double checking since I have gone back even further and can't seem to find a point where keyboard focus was set on initial render. If not a change in the repo, I'm not sure what else could be effecting this change in behavior.

Reply:
I've also tried running the above with `yarn` and got the same result.

Reply:
That sounds about right to me. And March 4 is the earliest I remember making changes to FluentTester directly so I guess it wasn't that :/

Reply:
@Saadnajmi sorry for getting back to this late: I chose FocusTrapZone because it was the easiest out of the two to get working in this scenario. The difference is that FTZ traps focus and FZ does not. FTZ was easier to work with because it placed focus automatically (w/o setting any props) whereas FZ would require some more props to be set.